crates/core/src/datasource/kafka/kafka_stream_read.rs

```diff
@@ -192,30 +191,9 @@ impl PartitionStream for KafkaStreamRead {
                     .await
                     {
                         Ok(Ok(m)) => {
-                            let timestamp = match m.timestamp() {
```

Comment:
This was dead code fwiw. We can add "add_kakfa_metadata" as an option to our source at some point.